src/main.cc

```diff
@@ -481,6 +481,7 @@ load_cfg(void)
 			       cfg_geti("flightrec_requests_size"),
 			       cfg_geti("flightrec_requests_max_req_size"),
 			       cfg_geti("flightrec_requests_max_res_size"));
+		say_set_on_log_level(flightrec_log_level());
```

Comment:
This had better be done from `flightrec_init()` - so as not to introduce yet another stub.

Reply:
Alongside with comment above - fixed.